src/tic.c

```diff
@@ -154,7 +155,7 @@ static inline u8 mapColor(tic_mem* tic, u8 color)
 
 static void setPixelDma(tic_mem* tic, s32 x, s32 y, u8 color)
 {
-	tic_tool_poke4(tic->ram.vram.screen.data, y * TIC80_WIDTH + x, mapColor(tic, color));
+	tic_tool_poke4(tic->ram.vram.screen.data, y * TIC80_WIDTH + x, color);
```

Comment:
pls restore `mapColor` function here, because of mapping affects not only sprites, it affects all the drawing api like `pix`, `line`, `rect` and etc...

Reply:
I moved the mapping function up into setPixel, and line, rect, etc. call the setPixel therefore it should be correct.